platform/configure/config.mdx

```diff
@@ -26,30 +30,30 @@ config:
       redirectURI: https://my-vcluster-host.com/auth/github/callback
 ```
 
-For a complete overview what helm values are possible, please take a look at the [Loft github repository](https://github.com/loft-sh/loft/tree/master/chart) directly.
+For a complete overview of possible Helm values, please refer to the [Loft GitHub repository](https://github.com/loft-sh/loft/tree/master/chart).
 
-### Changing the Loft Host variable
+### Changing the `loftHost` variable
```

Comment:
I would change this to `Setting a custom domain`. 

This is one of the situations where engineers love to talk about technical implementation vs. how this relates to what this means to the end user. 

User wants to set custom domain (that's how we should discuss it) but the technical implementation is the `loftHost` as that's what's in the values.